ml-gsd.d.ts

```diff
@@ -0,0 +1,157 @@
+export interface XYNumberArray {
```

Comment:
It doesn't make sense to create a type definitions file for a TypeScript project.